packages/blade/src/components/Box/__tests__/Box.native.test.tsx

```diff
@@ -111,4 +111,65 @@ describe('<Box />', () => {
     const { getByTestId } = renderWithTheme(<Box id={id} testID={testId} />);
     expect(getByTestId(testId)).toHaveProp('id', id);
   });
+
+  // https://github.com/razorpay/blade/issues/1624
+  it('should apply borderBottomColor prop to border bottom only', () => {
+    const { toJSON } = renderWithTheme(
+      <Box borderBottomColor="surface.border.normal.lowContrast">I am Visible</Box>,
+    );
+    expect(toJSON()).toMatchInlineSnapshot(`
```

Comment:
let's actually switch to external snapshots, otherwise will get too noisy here.

Reply:
Done.